src/components/CubeAnalysisNavbar.js

```diff
@@ -29,56 +14,39 @@ const CubeAnalysisNavBar = ({
   numCards,
   defaultFilterText,
 }) => {
-  const [isOpen, , toggleIsOpen] = useToggle(false);
-  const [filterCollapseOpen, , toggleFilterCollapseOpen] = useToggle(false);
-
-  let dropdownElement = <h5>Default Draft Format</h5>;
-
-  const dropdownCustomFormat = (format, formatIndex) => (
-    <DropdownItem
-      key={/* eslint-disable-line react/no-array-index-key */ `format-${formatIndex}`}
-      onClick={() => setFormatId(formatIndex)}
-    >
-      {format.title}
-    </DropdownItem>
-  );
+  const [navCollapseOpen, setNavCollapse] = useState(false);
```

Comment:
I set up the `useToggle` hook for exactly this - it's generally not good practice to pass callbacks to functions that aren't coming out of the `useCallback` memoized hook. It causes the javascript VM to do a bunch of unnecessary recompilation. Can you change these to useToggle as before and then pass the toggleNavCollapseOpen callback below?